src/goose/brokers/redis/console/pages/dead.clj

```diff
@@ -77,6 +79,27 @@
        (c/purge-confirmation-dialog data)
        [:button {:class "btn btn-danger btn-lg purge-dialog-show"} "Purge"]])]])
 
+(defn- job-page-view [{:keys       [base-path]
+                       {:keys [id]
+                        :as   job} :job}]
+  [:div.redis
+   [:h1 "Dead Job"]
+   (if job
+     [:div
+      [:form {:action (str base-path "/job/" id)
+              :method "post"}
+       [:div
+        (c/action-btns [(c/replay-btn {:disabled false})
+                        (c/delete-btn
+                          "Are you sure you want to the delete job?"
+                          {:disabled false})])
+        [:input {:name  "job"
+                 :type  "hidden"
+                 :value (utils/encode-to-str job)}]
+        (c/job-table job)]]]
+     (c/flash-msg {:type    :error
+                   :message "No job found"}))])
```

Comment:
```suggestion
                   :message "No jobs found"}))])
```

Reply:
@siripr4 since this is single job's view page I think, no job sounds better.

Reply:
`Job not found`, then? Either is fine actually
And I'm assuming it's too late to change this